Services/Music_Portal.Services.Services/MappingProfile.cs

```diff
@@ -1,8 +1,8 @@
 ﻿using AutoMapper;
-using Core;
-using Interfaces.Models;
+using Domain.Core;
+using Services.Interfaces.Models;
 
-namespace Services
+namespace Services.Services
```

Comment:
The namespace should start with the Project name and then internal folders and e.c. like
`namespace Music_Portal.Services.Services`